jest.config.js

```diff
@@ -35,6 +36,7 @@ export default {
     '<rootDir>/src',
   ],
   moduleNameMapper: {
+    '\\.(css|less)$': 'identity-obj-proxy',
```

Comment:
Why was this added?

Reply:
Some jest test were failing because jsdom css parser was not able to parse the stylesheet. I have fixed this issue by installing this package. These line are added to the Jest configuration file to ensure that our Jest test environment can properly handle imported CSS/LESS modules during testing. It is recommended in Jest documentation. https://jestjs.io/docs/webpack#mocking-css-modules